plugins/remmina/__init__.py

```diff
@@ -17,57 +19,72 @@
 __author__ = "Oğuzcan Küçükbayrak"
```

Comment:
I think you should add yourself as an author.

Reply:
Nah, you're the one to write it. I just made a simple improvement :)